tests/test_output.py

```diff
@@ -11,7 +11,7 @@
 
 from eko import basis_rotation as br
 from eko import output
-from ekobox import apply
+from ekomark import apply
```

Comment:
I know this is a bit tricky, but maybe can you find a way to split the apply tests (`test_apply` + `test_apply_flavor`) into a seperate file: `tests/ekomark/test_apply.py` ?

Reply:
Should be ok now